roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install" -r "{{ taca_ngi_dest }}/requirements.txt"
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: cd "{{ taca_ngi_dest }}" && "{{ ngi_pipeline_venv }}/bin/pip" install .
+  tags: taca
+
+-name: Create TACA delivery configs directory
+  file: path="{{ ngi_resources }}/TACA" state=directory mode=g+s
+  tags: taca
+
+- set_fact: 
+    site: "upps"
+  tags: taca
+
+- name: Deploy upps_taca_delivery.yml config
+  template: src="upps_taca_delivery.yml.j2" dest="{{ ngi_resources }}/TACA/{{ site }}_taca_delivery.yml"
```

Comment:
Some comment here about `ngi_resources` vs `ngi_pipeline_conf`. 

`upps_taca_delivery.yml.j2` should be `site_taca_delivery.yml.j2`.